.github/workflows/sonarcloud.yml

```
@@ -0,0 +1,94 @@
+                  distribution: "zulu"
+
+            - name: Cache SonarCloud packages
+              uses: actions/cache@v3
+              with:
+                  path: ~/.sonar/cache
+                  key: ${{ runner.os }}-sonar
+                  restore-keys: ${{ runner.os }}-sonar
+
+            - name: Cache SonarCloud scanner
+              id: cache-sonar-scanner
+              uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)